vendor/k8s.io/kubernetes/pkg/client/typed/discovery/restmapper.go

```diff
@@ -45,6 +45,10 @@ func NewRESTMapper(groupResources []*APIGroupResources, versionInterfaces meta.V
 	var resourcePriority []unversioned.GroupVersionResource
 	var kindPriority []unversioned.GroupVersionKind
 
+	// /v1 is special.  It should always come first
```

Comment:
```
// new servers will report non-core apigroups for origin resources. Because this client does not know about
// them yet, we enforce that all origin resources resolve to /v1 kinds.
```